specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2022-10-01-preview/settings.json

```diff
@@ -0,0 +1,313 @@
+          },
+          {
+            "name": "setting",
+            "description": "Setting to be created or updated.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Setting"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Setting Created.",
+            "schema": {
+              "$ref": "#/definitions/Setting"
+            }
+          },
+          "200": {
+            "description": "Setting Updated.",
+            "schema": {
+              "$ref": "#/definitions/Setting"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common-types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Settings"
+        ],
+        "operationId": "Settings_GetByScope",
+        "description": "Get the setting from the given scope by name.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/rest/api/cost-management/"
+        },
+        "x-ms-examples": {
+          "SettingByScope": {
+            "$ref": "./examples/setting-get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/scopeSettingParameter"
+          },
+          {
+            "$ref": "#/parameters/settingTypeParameter"
+          },
+          {
+            "$ref": "common-types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/Setting"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common-types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Settings"
+        ],
+        "operationId": "Settings_DeleteByScope",
+        "description": "Delete a setting within the given scope.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/rest/api/cost-management/"
+        },
+        "x-ms-examples": {
+          "SettingDeleteByScope": {
+            "$ref": "./examples/setting-delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/scopeSettingParameter"
+          },
+          {
+            "$ref": "#/parameters/settingTypeParameter"
+          },
+          {
+            "$ref": "common-types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded."
```

Comment:
>OK. The request has succeeded. [](http://example.com/codeflow?start=28&length=30)

Suggestion for the descriptions:
200 means: Settings Type is deleted successfully.
204 means: Settings Type is already deleted or not found.

Reply:
Thanks for the valuable feedback. we need to implement this change and deploy and then make this swagger update. I will create a task for this suggestion in our backlog.

Reply:
Thanks, Will implement as suggested